public/js/reader.js

```diff
@@ -331,7 +374,36 @@ Reader.updateMetadata = function () {
                 },
             });
         } else { $(".file-info").text(`${filename} :: ${width} x ${height} :: ${size} KB`); }
-    } else { $(".file-info").text(`Double-Page View :: ${width} x ${height}`); }
+    } else { 
+        // HEAD request to get filesize
+        let size = Reader.preloadedSizes[Reader.currentPage];
+        let size_pre = Reader.preloadedSizes[Reader.currentPage+1];
+        let size_view = size + size_pre;
+        if (!size || !size_pre) {
+            $.ajax({
```

Comment:
Since we're doing the HEAD call three times in this JS file now, it'd be better to refactor it to a specific function in `common.js` and call it here.